### PR TITLE
log to maintainers if plan loading takes too long

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

**Comment by dothebart** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/5518/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/6625/

File: arangod/Cluster/ClusterInfo.cpp [hunk 1]
```diff
@@ -474,6 +474,17 @@ void ClusterInfo::loadPlan() {
   ++_planProt.wantedVersion;  // Indicate that after *NOW* somebody has to
                               // reread from the agency!
 
+#ifdef ARANGODB_ENABLE_MAINTAINER_MODE
+  auto tStart = TRI_microtime();
+  auto longPlanWaitLogger = scopeGuard([&tStart]() {
+    auto tExit = TRI_microtime();
+    if (tExit - tStart > 0.5) {
+      LOG_TOPIC("66666", ERR, arangodb::Logger::FIXME) <<
```
Comment (jsteemann) [inline arangod/Cluster/ClusterInfo.cpp]:
```suggestion
      LOG_TOPIC("66666", WARN, Logger::CLUSTER) <<
```

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/8320/

**Comment by dothebart** [comment]
success except for the usual suspect.

**Pull request merged**